app/mesh/changelog.md

```diff
@@ -3,6 +3,38 @@ title: Kong Mesh Changelog
 no_version: true
 ---
 
+## 1.9.1
+> Release on **2022/10/10**
+
+Built on top of [Kuma 1.8.1](https://github.com/kumahq/kuma/releases/tag/1.8.1)
+
+### Changes
+
+- Gateway: add support for "retryOn"
+- Added support for evicted Pods
+- Support wildcard tag value match in RBAC
+- Deep copy tags when generate outbounds
```

Comment:
```suggestion
- When generating outbounds, tags are now a deep copy.
```
Is this accurate? It looks like "deep copy" can only be a noun.

Reply:
I copied this from the PR title and actually it doesn't make much value to users. It's better to say:
```suggestion
- Prevent a potential data race by creating a deep copy of tags when generating outbounds
```